compareECLFiles.cmake

```diff
@@ -196,6 +196,48 @@ function(add_test_compare_parallel_restarted_simulation)
                        PROPERTIES RUN_SERIAL 1)
 endfunction()
 
+
+###########################################################################
+# TEST: add_test_split_comm
+###########################################################################
+
+# Input:
+#   - casename: basename (no extension)
+#
+# Details:
+#   - This test class compares the output from a parallel simulation
+#     to that of a parallel simulation running with a custom communicator.
+function(add_test_split_comm)
+  set(oneValueArgs CASENAME FILENAME SIMULATOR ABS_TOL REL_TOL DIR)
+  set(multiValueArgs TEST_ARGS)
+  cmake_parse_arguments(PARAM "$" "${oneValueArgs}" "${multiValueArgs}" ${ARGN} )
+  if(NOT PARAM_DIR)
+    set(PARAM_DIR ${PARAM_CASENAME})
+  endif()
+  set(RESULT_PATH ${BASE_RESULT_PATH}/parallelRestart/${PARAM_SIMULATOR}+${PARAM_CASENAME})
```

Comment:
Is there a connection with restart? If not, we should probably use a different output directory.

Reply:
No connection, missed from copying, will amend.